crates/cli/Cargo.toml

```diff
@@ -10,9 +10,14 @@ path = "src/main.rs"
 
 [dependencies]
 tokio = { workspace = true }
+beerus-api = { path = "../api", optional = true }
 beerus-core = { path = "../core" }
 beerus-rpc = { path = "../rpc" }
```

Comment:
There must be a more expressive name than "api", especially when we already have a "rpc" crate.
If it takes care of tx execution, how about "execution"?

Reply:
Naming things is hard.

Reply:
Renamed to 'experimental-api`.